misc-scripts/generate_stable_ids.pl

```diff
@@ -234,7 +238,7 @@ sub get_highest_stable_id {
   }
 
   # Assuming that this is a correctly formatted stable id -> remove the
-  # G/T/P/E for exon etc.
+  # G/T/P for gene etc. (Exon dealt with above.)
 
   my ( $prefix, $suffix ) = $max =~ /([a-zA-Z]+)([0-9]+)/;
   if ( $type eq 'exon' ) {
```

Comment:
Could delete this to make the code more clear - execution will never reach this point if `$type eq 'exon'`

Reply:
Thanks, I've deleted that exon-based clause.